components/nav-link/nav-link.jsx

```diff
@@ -19,7 +19,7 @@ class NavLink extends React.Component {
   render() {
     return (
       <Link {...this.props}
-        className="open-sans"
+        className={`open-sans ${this.props.className}`}
```

Comment:
Problem was the `btn-add` class didn't get assigned after being passed to `<NavLink>`.